cartridges/int_klaviyo_core/cartridge/scripts/klaviyo/services.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/* API Includes */
+var Logger = require('dw/system/Logger');
+var ServiceRegistry = require('dw/svc/LocalServiceRegistry');
+var Site = require('dw/system/Site');
+
+
+// HTTP Services
+var KlaviyoEventService = ServiceRegistry.createService('KlaviyoEventService', {
+
+    /**
+   * Create the service request
+   * - Set request method to be the HTTP GET method
```

Comment:
We should update this to be `POST`

Reply:
the call itself is a post (set ~line 31) but this comment was not updated to reflect that.  we can update that comment.